### DISCUSSION TITLE
Can I use custom pipeline instead of default

### DISCUSSION THREAD

**Discussion by rajcspsg** [dream11/zio-http#717] [Q&A]
Hi,

This is the question - 

The examples for starting the server in this project always using [Server.make](https://github.com/dream11/zio-http/blob/main/zio-http/src/main/scala/zhttp/service/Server.scala#L101).
If I want to use some [custom http pipeline](https://github.com/rajcspsg/netty-mem-detect-demo/blob/master/core/src/main/java/com/myorg/netty/HttpServerPipelineFactory.java#L14-L22) , How can I achieve the same?

**Comment by tusharmath** [comment]
What's your use-case? Modification to the pipeline can introduce bugs and make the server unstable.
